ci/assets/large-x86.yaml

```diff
@@ -0,0 +1,54 @@
+cluster_name: large-x86
+
+provider:
+  type: aws
+  region: us-west-2
+  cache_stopped_nodes: false
+  security_group:
+    GroupName: ray-autoscaler-c1
+
+auth:
+  ssh_user: ubuntu
+  ssh_private_key: ~/.ssh/ci-github-actions-ray-cluster-key.pem
+
+max_workers: 8
+available_node_types:
+  ray.head.default:
+    resources: {"CPU": 0}
+    node_config:
+      KeyName: ci-github-actions-ray-cluster-key
+      InstanceType: i3.4xlarge
+      ImageId: ami-04dd23e62ed049936
+      IamInstanceProfile:
+        Name: ray-autoscaler-v1
+
+  ray.worker.default:
+    min_workers: 8
+    max_workers: 8
+    resources: {CPU: 100}
```

Comment:
Why limit to 100?

Reply:
Is there a way to specify minimum CPUs? I couldn't find that in the docs [here](https://docs.ray.io/en/latest/cluster/vms/references/ray-cluster-configuration.html#cluster-configuration-resources-type).